src/main/scala/org/apache/pekko/persistence/dynamodb/query/scaladsl/DynamoDBCurrentPersistenceIdsQuery.scala

```diff
@@ -69,7 +69,14 @@ trait CreatePersistenceIdsIndex {
 
 object CreatePersistenceIdsIndex {
 
-  /** required by [[DynamoDBCurrentPersistenceIdsQuery.currentPersistenceIdsByPageQuery]] */
+  /**
+   * required by [[DynamoDBCurrentPersistenceIdsQuery.currentPersistenceIdsByPageQuery]]
+   *
+   * When requesting snapshots by timestamp, Select ALL_ATTRIBUTES is used. Thus, a duplicate of
+   * the payload doesn't need to be stored in the index, which is more space efficient.
+   * This allows to make the choice between time and space efficiency by selecting the projection
+   * strategy for the created index.
```

Comment:
@spangaer could you make a small adjustment to this comment to say that the ALL_ATTRIBUTES change was only brought in v1.1.0?

Reply:
like that?